release-notes/5.0/preview/api-diff/preview2/.Net/5.0-preview2_System.Data.Common.md

```diff
@@ -0,0 +1,15 @@
+# System.Data.Common
+
+``` diff
+ namespace System.Data.Common {
+-    public abstract class DbCommand : Component, IAsyncDisposable, IDbCommand, IDisposable
++    public abstract class DbCommand : Component, IDbCommand, IDisposable
```

Comment:
We removed interfaces here?

Reply:
See https://github.com/dotnet/runtime/commit/d93109548c1924864d94bb612ac5fb848214e1f8

It's been fixed in P3